platform/linux-generic/_fdserver.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#ifdef HAVE_CONFIG_H
```

Comment:
https://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/acconfig-Header.html as I understand with that link we can add ifdef and endif on top and bottom of generated file.